src/main/schemas/facilities.schema.json

```diff
@@ -7,6 +7,25 @@
     "code" : {
         "type": "string",
         "pattern": "^[A-Z0-9]{2,3}$"
+    },
```

Comment:
I think the schema version should get updated in connection w/ this change.

Reply:
Roger that, will update.